Configure

```diff
@@ -1120,7 +1120,7 @@ $target{dso_extension}//=$target{shared_extension_simple};
 # Allow overriding the names of some tools.  USE WITH CARE
 # Note: only Unix cares about HASHBANGPERL...  that explains
 # the default string.
-$config{perl} =    ($^O ne "VMS" ? $^X : "perl");
+$config{perl} = env('PERL') || ($^O ne "VMS" ? $^X : "perl");
```

Comment:
This isn't quite the way to do it.  I'm submitting an alternative in a minute.

Reply:
See #6668

Reply:
OK, I'll take this out and use your solution.